test/test_cosmic_mutations.py

```diff
@@ -9,11 +10,11 @@
     FrameShiftTruncation,
 )
 
-annot = VariantAnnotator(75)
+ensembl = EnsemblRelease(75)
```

Comment:
It's used only as a default parameter, right? Otherwise it looks to me like we're declaring that this module uses a particular ensembl release